### ISSUE TITLE
Establishing first trustline end-to-end

### ISSUE THREAD

**Issue by Kisgus** [trustlines-network/feedback#39]
### Basic information:

* OS (Android / iOS / both):  Onboarding an Android user with iPhone
* Version: Newest

### Issue description:

**Context Step 0**

 Assuming the new user (user 1) is directed to "Overview" after signing up & shows the QR code from "Overview" to user 2)

**Step 1: User 2 scans the QR code and is directed to Trustline Request Flow**

This means skipping this screen 

![img_3232](https://user-images.githubusercontent.com/18598519/37459440-5b35e3cc-2848-11e8-84be-f5a1aff732eb.png)

And starting automatically on the "trustlines request" flow

![img_3231](https://user-images.githubusercontent.com/18598519/37459454-68449e0a-2848-11e8-8aef-12f9762116eb.png)

NB: User 2 is not sending any ETH to user 1 before actually "confirming" the trustline request

**Step 2: User 1 receives a trustline request on "Overview" as well as in the "Trustlines" page**

![screenshot_2018-03-15-11-34-38](https://user-images.githubusercontent.com/18598519/37459506-97ec650c-2848-11e8-9db8-d25c54dc64a5.png)

Currently this is only visible under "Overview"

Fix

- [x] The tutorial text here needs to be changed from:

*Click the creditline request and establish a trustline to start sending and receiving* 

to

*Click and review your first trustline request*

**Step 3: User 1 Review the trustline request**

Basically like below.

![screenshot_2018-03-15-11-35-53](https://user-images.githubusercontent.com/18598519/37459738-540b45dc-2849-11e8-8b08-6d225ec76961.png)

Fix:

- [x] Change name to "Trustline Request"

**Comment by Kisgus** [comment]
Solved, great!
